packages/devtools_app/lib/src/timeline/flutter/flutter_frames_chart.dart

```diff
@@ -62,8 +64,19 @@ class _FlutterFramesChartState extends State<FlutterFramesChart>
     super.didChangeDependencies();
     _controller = Controllers.of(context).timeline;
 
+    autoDispose(_controller.onTimelineCleared.listen((_) {
```

Comment:
call `cancel()` to clear out all existing autoDispose subscriptions before adding any new nodes.

Reply:
done